Operational-Procedures/Troubleshooting/component-offline.rst

```diff
@@ -0,0 +1,148 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Michael Reuter, Patrick Ingraham*
+
+.. _troubleshooting-component-offline:
+
+#################
+Component Offline
+#################
+
+.. warning::
+
+    This procedure contains steps that may affect the system in crical ways.
+
+    If you are unsure about any of the steps described here, contact support personnel.
+
+
+* Most (but not all) components in the Rubin Observatory Control System, when sent to OFFLINE will terminate its execution process and will no longer be available in the system.
```

Comment:
Change to: * Most components in the Rubin Observatory Control System, when sent to OFFLINE, will terminate its execution process and will no longer be available in the system.